roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-config.j2

```diff
@@ -62,6 +62,9 @@ use-octavia={{ external_openstack_lbaas_use_octavia }}
 lb-provider=octavia
 use-octavia=true
 {% endif %}
+{% if external_openstack_enable_ingress_hostname is defined %}
+enable-ingress-hostname={{ external_openstack_enable_ingress_hostname | bool }}
+{% endif %}
```

Comment:
Could you add the default value of this option into roles/kubespray-defaults/defaults/main.yaml ?
We can see default values of external_openstack_* on the file like
```
# Default values for the external OpenStack Cloud Controller
external_openstack_lbaas_create_monitor: false
external_openstack_lbaas_monitor_delay: "1m"
external_openstack_lbaas_monitor_timeout: "30s"
external_openstack_lbaas_monitor_max_retries: "3" 
external_openstack_network_ipv6_disabled: false
external_openstack_lbaas_use_octavia: false
external_openstack_network_internal_networks: []
external_openstack_network_public_networks: []
```

Reply:
Good point @oomichi. Added.